public/app/features/commandPalette/useMatches.ts

```diff
@@ -1,8 +1,9 @@
-import uFuzzy from '@leeoniya/ufuzzy';
 import { ActionImpl, Priority, useKBar } from 'kbar';
 import { useThrottledValue } from 'kbar/lib/utils';
 import * as React from 'react';
 
+import { fuzzySearch } from '@grafana/data';
+
 // From https://github.dev/timc1/kbar/blob/main/src/useMatches.tsx
 // TODO: Go back to useMatches from kbar when https://github.com/timc1/kbar/issues/255 is fixed
```

Comment:
instead of a todo, maybe replace it with a note about we are NOT using useMatches?